Gruntfile.js

```diff
@@ -166,6 +166,14 @@ module.exports = function (grunt) {
       }
     },
 
+    // ESLint is a replacement for JSHint + includes style linting + is extensible
+    eslint: {
+        target: [
+          'Gruntfile.js',
+          '<%= yeoman.app %>/scripts/{,*/}*.js'
+        ]
+    },
```

Comment:
I'd hope we could get a config that would only enforce new rules on newly changed files...